ingestion-beam/src/main/java/com/mozilla/telemetry/util/Json.java

```diff
@@ -201,11 +202,17 @@ public static String asString(Object data) throws IOException {
    * {@code getAttributeMap} method returns the value for the {@code attributes} parameter.
    * Additionally jackson doesn't like that there are no setter methods on {@link PubsubMessage}.
    *
+   * <p>Beam 2.16 added the {@code messageId} field, which is not relevant for our purposes;
+   * we mark that field with {@link JsonIgnore} so that it is not serialized and we maintain
+   * compatibility with existing messages that our pipeline has persisted without {@code messageId}.
```

Comment:
Most of our tests were failing due to `"messageId":null` newly appearing in all the serialized JSON blobs.